stl/inc/xstring

```diff
@@ -2642,6 +2642,47 @@ private:
     }
 
 public:
+#if _HAS_CXX20
+    void _Move_assign_from_buffer(_Elem* const _Right, const size_type _Size, const size_type _Res) {
+        // Move assign from a buffer, used by basic_stringbuf
+        _Tidy_deallocate();

```

Comment:
I don't think you are correctly allocating a new debug proxy after this deallocation. (i.e. we are at least missing a `_Orphan_all` somewhere)

Reply:
`_Orphan_all` is in `_Tidy_deallocate`.

Reply:
I see.